server.R

```diff
@@ -20,6 +20,18 @@ library( viridis )
 
 ####  Server  ####
 server <- function( input, output, session ) {
+
+  # store selected variable
+  user.variable <- reactive({
+    
+    table.selected <- if_else(input$selectTableSlim == "Other", input$selectTable, input$selectTableSlim)
```

Comment:
`table.selected` should be its own `reactive()` expression. It is used throughout the server so store it once and simply call `table.select()` throughout to avoid repeating this `if_else()` call.

Reply:
Nooooooooooooooooow I get it!